server/session/player.go

```diff
@@ -328,6 +329,57 @@ func (s *Session) SendForm(f form.Form) {
 	})
 }
 
+// SendDialogue shows a npc dialogue to the client of the connection. The Submit method of the dialogue is called when
+// the client clicks a button or closes the menu.
+func (s *Session) SendDialogue(d dialogue.Dialogue) {
+	h := s.handlers[packet.IDNPCRequest].(*NpcRequestHandler)
+
+	m := d.Menu()
+	h.mu.Lock()
+	h.dialogues[s.c.XUID()] = d
+	h.mu.Unlock()
+
+	s.entityMutex.Lock()
+	RID := s.entityRuntimeIDs[m.NPC()]
```

Comment:
Use `(*Session).entityRuntimeID`

Reply:
you you mean currentEnttiyRunetimeId() ?